lib/request-wrapper.ts

```diff
@@ -107,6 +93,13 @@ export class RequestWrapper {
 
     this.axiosInstance = axios.create(axiosConfig);
 
+    // axios sets the default Content-Type for `post`, `put`, and `patch` operations
+    // to 'application/x-www-form-urlencoded'. This causes problems, so overriding the
+    // defaults here
+    ['post', 'put', 'patch'].forEach((op) => {
```

Comment:
I assume these defaults are only used if the request does not contain the Content-Type header, but we'd expect Content-Type to be set on all requests invoked via an SDK (that have a request body).  Is that right?

Reply:
Yes, that's correct. Axios just wants to use `application/x-www-form-urlencoded` as the default and `application/json` is a better fallback assumption. I don't remember what the scenario was that involved the default being used and us discovering this problem but the solution has been in the code for a while